packages/css/src/components/selectable/selectable.template.ts

```diff
@@ -33,6 +34,10 @@ export function selectableTemplate({
         ?required=${required}
         ?checked=${checked}
       >
+      ${indeterminate
+        ? html`<script>document.getElementById('${id}').indeterminate = true;</script>`
```

Comment:
Werkt 

```
html`<script>document.getElementById('${id}').indeterminate = ${indeterminate};
```

niet?

Reply:
En moet ie expliciet op `false` gezet worden? Is dat voor Storybook waar indeterminate eerst op `true` is gezet?

Reply:
Volgens mij had dat iets met chaching en/of reactivness te maken ofzo, maar ik heb nog geen andere (ogenschijnlijk betere) code gevonden die wel doet wat het moet doen.

Reply:
Het lampje gaat weer branden... het heeft geen zin om na het laden van de pagina het script aan te passen d.m.v. controls, want dan is het script al uitgevoerd. Dit werkt alleen (een beetje) omdat/nadat `&args=indeterminate:false` in de url terecht komt, want dan wordt het juiste script uitgevoerd.

Reply:
Dat is een jammertje in Storybook, dat ook bij andere boolean controls optreedt.